core/src/com/fourinachamber/fortyfive/rendering/BetterShader.kt

```diff
@@ -52,6 +55,14 @@ class BetterShader(
             )
         }
 
+        "u_perlin512x512" -> {
+            // TODO: slightly ugly
```

Comment:
if we start like that, we would need to start making definitions for what is what and which types of "ugly" exists. i would recommend at least: "close to not ugly", "sligtly ugly", "ugly", "very ugly", "godly ugly" and last but not least "this code should have never been written"